Gemfile.lock

```diff
@@ -149,7 +149,7 @@ GEM
     faraday (1.1.0)
       multipart-post (>= 1.2, < 3)
       ruby2_keywords
-    ffi (1.11.3)
+    ffi (1.15.3)
```

Comment:
in general, we shouldn't upgrade gems as part of other tickets

Reply:
When I did `bundle install` I have to update this gem in order to work. You think I have to remove this file from PR?

Reply:
we can leave it this time